doc/manual/checker_library.md

```diff
@@ -85,3 +85,52 @@ traceability). The metadata produced by a check shall be included and
 described in its documentation, (e.g., if a check logs in metadata version and
 configuration of a simulator, the checker documentation shall describe the
 content of aforementioned metadata fields)._
+
+## Manifest File
+
+Each Checker Library needs to provide a manifest file with configuration
+information for the framework. The format of this file is aligned with the one
+for the [Result File (`*.xqar`)](file_formats.md#result-file-xqar) and follows
+the same specification. The only difference is that it may contain additional
+Report Modules, which are delivered as part of the Checker Library.
+
+### Example
+
+```xml
+<CheckerResults version="1.0.0">
+    <CheckerBundle name="CheckerBundle1" description="Description of the Checker Bundle" build_date="2024-03-26" version="1.0.0">
+        <Executable name="/usr/bin/python3"> <!-- apply to Linux only! Probably just "python"? -->
+            <Arg>my_fancy_script.py</Arg> <!-- Mandatory arguments, followed by the configuration.xml or input file-->
+            <Arg>--fancy-param=True</Arg>
```

Comment:
Info about the current implementation: https://github.com/asam-ev/qc-framework/blob/main/doc/manual/writing_user_defined_modules.md 

There are three ways to call a Checker Bundle
* Start with configuration file (official approach from the framework)
* Create "empty result file" --> this is the manifest for the Configuration GUI, see also: https://github.com/asam-ev/qc-framework/blob/main/doc/manual/using_the_framework.md#add-self-implemented-checkerbundles-and-reportmodules
* Start with default configuration (without configuration file, for simple batch processing, when called without the framework. I did that many times when showing demo, calling `CheckXodr.bat`)

Reply:
I would go with placeholder arguments, such as `<Arg>{CONFIG}</Arg>`. We just need to document them, and eventually include in schema the presence for them when we consider them mandatory.

Reply:
I mean if the placeholder / resolvable aguments just resemble the same as, e.g., `<Param name="CheckerBundleParameter" value="0.1"/>` , I would not include them since it will be unclear where to derive the current configuration from (CLI vs File).
My understand was more, that the `<Arg>` will represent executable specific arguments like `main.py` for an interpreted Python file.

But as @andreaskern74 mentioned in the docs, there is a way to call the checker only via CLI args: [Check a Single File With One Single CheckerBundle
](https://github.com/asam-ev/qc-framework/blob/main/doc/manual/writing_user_defined_modules.md#check-a-single-file-with-one-single-checkerbundle) So probably, the placeholder stuff will be necessary.



However, while checking the docs I came across another "difficulty":
```
> checker_bundle_exe configuration.xml
```

```
> checker_bundle_exe track.xodr [-o1 -o2 ...]
```
Both calls will look from the checkers perspectiv identically. Determining whether the specified parameter is a configuration or the file to be checked is therefore more difficult than it should be. I know reading the file and determining if it is a configuration is probably not that hard, but still it would be much easier if, we just introduce a slightly different call for each version of these, e.g.,:
```
> checker_bundle_exe -c configuration.xml
> checker_bundle_exe --config configuration.xml
```
or
```
> checker_bundle_exe --check track.xodr [-o1 -o2 ...]
```
However, this discussion should be probably shifted to another place.

Reply:
> However, this discussion should be probably shifted to another place.

👍